web/web/static/web/html/panes/panes.html

```diff
@@ -0,0 +1,24 @@
+<div class="panesContainer">
```

Comment:
Looks like the css for class "panesController" needs to be applied to the panes tag instead of this one. the "overflow: hidden" css doesn't seem to be applied to the panes properly and they don't fit to screen like they used to.